collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="@drawable/project_settings_dialog_background"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/close_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_close"
+        android:background="?attr/selectableItemBackgroundBorderless"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/projects"
+        style="@style/TextAppearance.Collect.Subtitle1"
+        app:layout_constraintBottom_toBottomOf="@+id/close_icon"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/close_icon" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/general_settings_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+        android:layout_marginTop="@dimen/margin"
+        android:text="@string/general_preferences"
+        android:textColor="@color/deep_purple_500"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/close_icon"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/admin_settings_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+        android:layout_marginTop="@dimen/margin_small"
+        android:text="@string/admin_preferences"
+        android:textColor="@color/deep_purple_500"
```

Comment:
I thought these should really be black (`colorOnSurface`) but that also requires customization. We can come back to the color and style here when we've got more of the dialog in place.

Reply:
Yeah let's leave it as-is now and discuss later.